.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
         additional_dependencies: [toml]
         # need to exclude files here as isort in pre-commit hook doesn't seem
         # to look for config in pyproject.toml
-        exclude: (rokku\.py|raspberry_pi_camera\/tests\/|raspberry_pi_motion_sensor\/tests\/|raspberry_pi_alarm\/tests\/|raspberry_pi_ui\/utility\.py|raspberry_pi_intercom\/mumble\.py)
+        exclude: (rokku\.py|raspberry_pi_camera\/tests\/|raspberry_pi_motion_sensor\/tests\/|raspberry_pi_alarm\/tests\/|raspberry_pi_ui\/utility\.py|raspberry_pi_intercom\/mumble\.py|embedded_yt\.py)
```

Comment:
Good! But there is another place you will have to place the excluded file. Inside `pyproject.toml`, under `[tool.isort]`, in the list of `skip`, add `"src/raspberry_pi_ui/embedded_yt.py"`